opentech/apply/funds/api_views.py

```diff
@@ -188,3 +191,38 @@ def perform_create(self, serializer):
             submission=obj.submission,
             related=obj,
         )
+
+
+class CommentEdit(
+        mixins.RetrieveModelMixin,
+        mixins.CreateModelMixin,
+        generics.GenericAPIView,
+):
+    queryset = Activity.comments.all()
+    serializer_class = CommentEditSerializer
+    permission_classes = (
+        permissions.IsAuthenticated, IsAuthor
+    )
+
+    def post(self, request, *args, **kwargs):
+        return self.edit(request, *args, **kwargs)
+
+    @transaction.atomic
+    def edit(self, request, *args, **kwargs):
+        comment_to_edit = self.get_object()
+        comment_to_update = self.get_object()
+
+        comment_to_edit.previous = comment_to_update
+        comment_to_edit.pk = None
+        comment_to_edit.edited = timezone.now()
+
+        serializer = self.get_serializer(comment_to_edit, data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        if serializer.validated_data['message'] != comment_to_update.message:
+            self.perform_create(serializer)
```

Comment:
Without testing this locally, I am a uncertain whether this will add a messenger entry as in `CommentListCreate`

Reply:
This definitely doesn't.
We spoke offline -- was not specced. Could be beneficial, but may get noisy with small spelling edits (for example)